ietf/templates/meeting/agenda.html

```diff
@@ -58,8 +58,10 @@ <h2>
         </p>
     {% endif %}
     {% include 'meeting/tz-display.html' with meeting_timezone=timezone only %}
-    {% include "meeting/agenda_filter.html" with filter_categories=filter_categories customize_button_text="Personalize the agenda view..." always_show=personalize %}
-    {% include "meeting/agenda_personalize_buttonlist.html" with meeting=schedule.meeting only %}
+    {% include "meeting/agenda_filter.html" with filter_categories=filter_categories customize_button_text="Filter this agenda view..." always_show=personalize %}
+    {% if personalize %}
```

Comment:
No - this isn't right - The buttons are _supposed_ to be on the main agenda page as well as the other page, as they are pre-bs5.
After someone customizes with either mechanic, they should be able to easily subscribe or download ics.

Reply:
Yeah, I lost track of which buttons these were. It's just the link back to the customized agenda view that needed to be limited. They all needed to be made visible on the personalize tab. Thanks for catching it.